code/modules/reagents/machinery/mortar.dm

```diff
@@ -70,9 +24,11 @@
 	if (crushable)
 		to_chat(user, "<span class ='warning'>There's already something inside!</span>")
 		return 1
-	if (!is_type_in_list(O, blend_items) && !is_type_in_list(O, juice_items))
-		to_chat(user, "<span class ='warning'>You can't grind that!</span>")
-		return ..()
+
+	//There used to be a check here to only allow certain whitelisted things to be put inside the mortar. Now we just check size. God help us.
```

Comment:
By your logic it's hardcoding that only items can be picked up. Let's just dumb cast every object into item slots!

Having a giant list of things is hardcoding. Having functionality so you can't put everything into a grinder in an OOP fashion is not.